README.md

```diff
@@ -81,9 +83,14 @@ Getting started
     end
     ```
 
-3. Run your tests, open up `coverage/index.html` in your browser and check out
-   what you've missed so far.
-4. Add the following to your `.gitignore` file to ensure that coverage results
+3. Run your full test suite to see the percent coverage that your application has.
+4. After running your tests, run the command below from your app's root directory to check out what parts of your
+   application need more tests:
+
+  ```
+  open coverage/index.html
```

Comment:
This command only works on mac, which is why we didn't include it earlier. I think I'd prefer something like `open `coverage/index.html` in the browser of your choice` or something. If you think it's helpful you can include the sample but please clearly mark it as mac specific code :)

Reply:
Oh, awesome. Makes sense. I'll take care of that and send it over. Let me know if there's anything else needing a change once I send it over! I know it's a small fix, but I love this gem, and wanted to contribute in a minor way!